app/models/loan_filtered_question.rb

```diff
@@ -2,6 +2,10 @@
 class LoanFilteredQuestion < FilteredQuestion
   attr_accessor :loan
 
+  # the super initialize is breaking this in the tests since it requires a division
+  # I'm thinking two things. At the moment, preferring 1
+    # create a separate class that inherits from filtered_questions for division
+    # update this initialize method, the tests for it and all other usages with division arg
```

Comment:
Actually, this subclass will need to pass in a division anyway. We can leave it failing for the moment.

Reply:
alrighty